test/python/test_rticonnextdds_data_access.py

```diff
@@ -156,7 +170,7 @@ def test_bad_member_name(self, populated_input):
   def test_bad_member_syntax(self, populated_input):
     sample = populated_input[0]
     with pytest.raises(rti.Error) as execinfo:
-      sample.get_number("my_point_sequence[10[.y")
+      sample.get_number("my_point_sequence[9[.y")
```

Comment:
typo (since we are testing for failure the test still passes)

Reply:
It's not a typo. The expected failure is the bad bracket.